airflow-core/src/airflow/utils/log/file_task_handler.py

```diff
@@ -166,17 +242,133 @@ def _parse_log_lines(
                 log.timestamp = coerce_datetime(log.timestamp)
                 timestamp = log.timestamp
             yield timestamp, idx, log
+        idx += 1
 
 
-def _interleave_logs(*logs: str | LogMessages) -> Iterable[StructuredLogMessage]:
-    min_date = pendulum.datetime(2000, 1, 1)
+def _sort_key(timestamp: datetime | None, line_num: int) -> int:
```

Comment:
```suggestion
def _create_sort_key(timestamp: datetime | None, line_num: int) -> int:
```

not sure whether it's better 🤔 it kinda confused me when I saw `sort_key` is a none and `_sort_key` looks like a verb that returns `sort_key`

Reply:
Agree, adding the verb for prefix is more readable